main.go

```diff
@@ -84,10 +91,13 @@ func main() {
 	watchNamespace, _ := os.LookupEnv(watchNamespaceEnvVar)
 
 	controllerOptions := ctrl.Options{
-		Namespace:              watchNamespace,
-		Scheme:                 scheme,
-		MetricsBindAddress:     metricsAddr,
-		Port:                   9443,
+		Scheme: scheme,
+		Metrics: metricsserver.Options{
+			BindAddress: metricsAddr,
+		},
+		WebhookServer: webhook.NewServer(webhook.Options{
+			Port: 9443,
+		}),
```

Comment:
Do we have to enable the webhook server?
We don't serve any webhooks and there is no reason to enable it unless we do.

Reply:
I was wondering the same, but kept it since it was originally defined in
```
controllerOptions := ctrl.Options{
  Namespace:              watchNamespace,
  Scheme:                 scheme,
  MetricsBindAddress:     metricsAddr,
  Port:                   9443,
```
So this was to just maintain the configuration option with the dependency update

Reply:
I'm indifferent to it's existence though, and +1 to removing it if we don't use it, but perhaps this should be a separate commit to have a clear history of it being removed

Reply:
Sounds reasonable, I guess it just got clearer to me what Port actually did now that it said webhook first :)